yarn-project/end-to-end/src/e2e_fees/fees_test.ts

```diff
@@ -135,6 +134,7 @@ export class FeesTest {
       async ({ accountKeys }, { pxe, aztecNode, aztecNodeConfig }) => {
         this.pxe = pxe;
         this.aztecNode = aztecNode;
+        this.gasSettings = GasSettings.default({ maxFeesPerGas: await this.aztecNode.getCurrentBaseFees() });
```

Comment:
I assumed that these tests sent few enough txs that prices would not spike. Would it be safer to set these fees to twice the current base fees, just in case?